src/api/goods.ts

```diff
@@ -6,6 +6,16 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => {
+  return fetch(API_URL)
+    .then(response => response.json())
+    .then(result => (
+      result.sort((a: Good, b: Good) => a.name.localeCompare(b.name))
```

Comment:
```suggestion
  return getAll()
    .then(result => (
      result.sort((a: Good, b: Good) => a.name.localeCompare(b.name))
```
Можна тут перевикористовувати функцію getAll